home.html

```diff
@@ -126,72 +126,186 @@
-      <div class="row">
-        <div class="col-md-3">
-          <div
-            id="carousel-pager"
-            class="carousel slide"
-            data-ride="carousel"
-            data-interval="500000000"
-          >
-            <!-- Carousel items -->
-            <div class="carousel-inner vertical">
-              <div class="active item">
-                <img
-                  src="http://placehold.it/600/f44336/000000&amp;text=First+Slide"
-                  class="img-responsive"
-                  data-target="#carousel-main"
-                  data-slide-to="0"
-                />
+    <div class="row m-4">
+      <!-- SECTION_ONE -->
+      <div class="col-2 mt-4">
+        <div class="carousel-inner vertical mt-4">
+          <div class="active item">
+            <img
+              src="asset/milk.png"
+              class="vertical_slider_height_width"
+              data-target="#carousel-main"
+              data-slide-to="0"
+            />
+          </div>
+          <div class="active item">
+            <img
+              src="asset/milk.png"
+              class="vertical_slider_height_width"
+              data-target="#carousel-main"
+              data-slide-to="0"
+            />
+          </div>
+          <div class="active item">
+            <img
+              src="asset/milk.png"
+              class="vertical_slider_height_width"
+              data-target="#carousel-main"
+              data-slide-to="0"
+            />
+          </div>
+          <div class="active item">
+            <img
+              src="asset/milk.png"
+              class="vertical_slider_height_width"
+              data-target="#carousel-main"
+              data-slide-to="0"
+            />
+          </div>
+          <div class="active item">
+            <img
+              src="asset/milk.png"
+              class="vertical_slider_height_width"
+              data-target="#carousel-main"
+              data-slide-to="0"
+            />
+          </div>
+          <div class="active item">
+            <img
+              src="asset/milk.png"
+              class="vertical_slider_height_width"
+              data-target="#carousel-main"
+              data-slide-to="0"
+            />
+          </div>
+          <div class="active item">
+            <img
+              src="asset/sc31.jpg"
+              class="vertical_slider_height_width"
+              data-target="#carousel-main"
+              data-slide-to="0"
+            />
+          </div>
+        </div>
+      </div>
+
+      <!-- SECTION_TWO -->
+      <div class="col-5">
+        <h3 class="blue_text_color">
+          AGELESS MULTI-COLLAGEN® PROTEIN POWDER (4 DELICIOUS FLAVORS)
+        </h3>
+        <div class="d-flex align-items-center">
+          <img
+            src="asset/featured_review_stars_small.png"
+            class="start_img"
+            alt=""
+          />
+          <p class="review mt-3">2909 Reviews</p>

```

Comment:
Give proper name to classes.This review class is giving a color. Is it not possible to use bootstrap class here ?